src/controllers/dynakube/dynakube_controller_test.go

```diff
@@ -186,7 +215,7 @@ func TestReconcileActiveGate_Reconcile(t *testing.T) {
 					Value:     "https://proxy:1234",
 					ValueFrom: "",
 				}}}
-		controller := createFakeClientAndReconcile(mockClient, instance, testPaasToken, testAPIToken)
+		controller := createFakeClientAndReconciler(mockClient, instance, testPaasToken, testAPIToken)
```

Comment:
there should also be a test which checks that the mock client is called with the set label